Directory.Build.props

```diff
@@ -8,21 +9,9 @@
     <DebugType>embedded</DebugType>
     <DebugSymbols>true</DebugSymbols>
     <LangVersion>Latest</LangVersion>
-    <IsPackable>true</IsPackable>
```

Comment:
I guess this was redundant for all the assemblies that will reside in this repo.

Reply:
Yes, I think it's better to define this property in the project file and set to false when the package shouldn't be produced in a specific release.